google-cloud-vision/acceptance/google/cloud/vision/v1p1beta1/image_annotator_smoke_test.rb

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# AUTO GENERATED BY GAPIC
+
+require "minitest/autorun"
+require "minitest/spec"
+
+require "google/cloud/vision"
+
+describe "ImageAnnotatorSmokeTest v1p1beta1" do
+  it "runs one smoke test with batch_annotate_images" do
+
+    image_annotator_client = Google::Cloud::Vision.new(version: :v1p1beta1)
```

Comment:
This is going to fail. The constructor is now `Vision::ImageAnnotator.new`, not `Vision.new`.